src/sentry/eventstore/models.py

```diff
@@ -80,7 +80,7 @@ def message(self):
         if column in self._snuba_data:
             return self._snuba_data[column]
 
-        return self.data.get("message")
+        return self.real_message
```

Comment:
This came up in a few different PRs, we need to make sure we are returning the actual message string regardless of where it's loaded from instead of the dict that is in nodestore